src/wlapplication_messages.cc

```diff
@@ -19,178 +19,273 @@
 
 #include "wlapplication_messages.h"
 
+#include <algorithm>
 #include <iostream>
+#include <regex>
 
 #include <boost/format.hpp>
 
 #include "base/i18n.h"
 
-using std::endl;
+constexpr size_t kIndent = 23;
+constexpr size_t kTextWidth = 50;
+
+#ifndef _WIN32
+static const std::string kDefaultHomedir = "~/.widelands";
+#else
+static const std::string kDefaultHomedir = "%USERPROFILE%\\.widelands";
+#endif
+
+/// Command line help
+/// Title: unindented text in the line above
+/// Key: the actual parameter
+/// Hint: text after =
+/// Help: Full text help
+/// Verbose: Filter some config options (--help vs. --help-all)
+static std::vector<Parameter> parameters;
+void fill_parameter_vector() {
+	i18n::Textdomain textdomain("widelands_console");
+	parameters =
+	{ {_("Usage:"), _("widelands <option0>=<value0> ... <optionN>=<valueN>"), "--", "", false},
+	  {"", _("widelands <save.wgf>/<replay.wrpl>"), "--", "", false},
+	  {_("Options:"), _("<config-entry-name>"), _("value"),
+		_("value overwrites any config file setting Note: New value will be written to config file"),
```

Comment:
```suggestion
		_("`value` overwrites any config file setting. Note: New value will be written to config file"),
```

Reply:
The '.' between the two sentences is still missing

Reply:
Sorry, overlooked that '.' Should be fixed now.